company-kata-solutions/src/test/java/org/eclipse/collections/companykata/Exercise8Test.java

```diff
@@ -165,7 +175,12 @@ public void ordersByCustomerUsingAsMap()
     @Test
     public void mostExpensiveItem()
     {
-        MutableListMultimap<Double, Customer> multimap = null;
+        Function<Customer, Double> mostExpensiveItem = customer -> customer.getOrders()
+            .flatCollect(Order::getLineItems).collect(LineItem::getValue)
+            .sortThis(Collections.reverseOrder()).getFirstOptional().orElse(Double.MIN_VALUE);
+
+        MutableListMultimap<Double, Customer> multimap = this.company.getCustomers()
+            .groupByAndCollect(mostExpensiveItem, customer -> customer, Multimaps.mutable.list.empty());
```

Comment:
Requesting the same formatting change here as well.

Reply:
Done